src/authMachine.ts

```diff
@@ -165,7 +166,10 @@ export const createAuthMachine = (params: {
               src: (_): Promise<LoggedInUser> =>
                 callAPI<LoggedInUser>({
                   endpoint: 'get-user',
-                }).then((res) => res.data),
+                }).then((res) => {
+                  analytics()?.identify(res.data.id);
```

Comment:
This one feels to me more like an action called in response to this async job - it's not exactly a core part of it. I'd move it to `onDone.actions`

Reply:
Good idea. Now that this is working, generally, I can move it to `onDone`.